pages/osx/brightness.md

```diff
@@ -0,0 +1,15 @@
+# brightness
+
+> Set and obtain the brightness level of all internal and certain external displays.
```

Comment:
Maybe "Get" instead of obtain ?

So then it should be - "Get and set .."

Reply:
Sounds easier to understand which is desirable ^^